mesonpy/__init__.py

```diff
@@ -683,7 +683,12 @@ def __init__(
                 if other:
                     raise ConfigError(f'Multi-architecture builds are not supported but $ARCHFLAGS={archflags!r}')
 
-                macver, _, nativearch = platform.mac_ver()
+                if sysconfig.get_platform().endswith('-ppc64'):
```

Comment:
`sysconfig.get_platform()` may return `fat`, `fat3`, `fat64`, or `universal` for multi-architecture Python binaries with ppc or ppc64 support.

Reply:
Ah, I know what you refer to. If it uses those values, it is not a suitable choice.
(On a side note and ironically, universal build of Python itself is broken for PowerPC: there is no combo of ppc+ppc64, but this is unrelated as such to the discussed problem.)

What could you suggest to make it work correctly?